uat/testing-features/src/main/java/com/aws/greengrass/steps/MqttControlSteps.java

```diff
@@ -333,6 +333,26 @@ public void connect(String clientDeviceId, String componentId, String brokerId,
         throw lastException;
     }
 
+    /**
+     * Creates MQTT connection.
+     *
+     * @param clientDeviceId the id of the device (thing name) as defined by user in scenario
+     * @param componentId  the componentId of MQTT client
+     * @param brokerId the id of broker, before must be discovered or added by default
+     * @param mqttVersion the MQTT version string
+     * @throws RuntimeException throws in fail case
+     *
+     */
+    @SuppressWarnings({"PMD.UseObjectForClearerAPI", "PMD.AvoidCatchingGenericException"})
+    @And("I can not connect device {string} on {word} to {string} using mqtt {string}")
+    public void canNotConnect(String clientDeviceId, String componentId, String brokerId, String mqttVersion) {
+        try {
+            connect(clientDeviceId, componentId, brokerId, mqttVersion);
```

Comment:
That code do not check connection is not established due to always passed

Reply:
fixed in 7d9ef728e6680002c72c9f03579e66403cd6a829